### PR TITLE
Update the user cookie when confirming an email

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- api

### CODE REVIEW

File: api/routes/api/email.js [hunk 1]
```diff
@@ -190,8 +190,19 @@ emailRouter.get('/validate', (req, res) => {
...
+      req.login(user, err => {
+        if (err) {
+          return res
+            .status(400)
+            .send(
+              'We ran into an issue validating this email address. You can re-enter your email address in your community settings to resend a confirmation email, or get in touch with us at hi@spectrum.chat.'
+            );
+        }
+
+        if (!user.username) return res.redirect(rootRedirect);
```
Comment (mxstbr) [inline api/routes/api/email.js:203]: Why not just redirect to `/users/me/settings` across the board? No need to check for a username here?

Comment (brianlovin) [reply api/routes/api/email.js:203]: Because that redirect doesn't work if there's no username. Tested locally. It takes users to a signin view.

Comment (mxstbr) [reply api/routes/api/email.js:203]: Oh, okay!

**Review by mxstbr — DISMISSED** [review]
If it works let's ship it

**Review by mxstbr — APPROVED** [review]

**Pull request merged**